exonum-java-binding/qa-service/src/test/java/com/exonum/binding/qaservice/transactions/ThrowingTxTest.java

```diff
@@ -77,7 +77,7 @@ but also to access the description and match it against a condition (e.g., conta
     ExecutionStatus txResult = blockchain.getTxResult(throwingTx.hash()).get();
     assertTrue(txResult.hasError());
     ExecutionError error = txResult.getError();
-    assertThat(error.getKind()).isEqualTo(ErrorKind.RUNTIME);
+    assertThat(error.getKind()).isEqualTo(ErrorKind.SERVICE);
```

Comment:
Why SERVICE if SERVICE means service-defined errors? What will be the error code? Will it clash with service-defined error-codes?

Reply:
I do not necessarily object to representing 'unexpected' errors with SERVICE, but it requires a predefined and documented (and forbidden to be used explicitly ) error code to represent those.

Reply:
I am in favour of redefining SERVICE as any errors *occurring* in service transactions instead of service-*defined*.

Reply:
Was my mistake, runtime execution error got invalid error kind.